Source/SIMPLib/Geometry/RectGridGeom.h

```diff
@@ -56,10 +56,10 @@ class SIMPLib_EXPORT RectGridGeom : public IGeometryGrid
      * @return
      */
     static Pointer CreateGeometry(const QString& name);
-    
-    void setXBounds(FloatArrayType::Pointer xBounds);
-    void setYBounds(FloatArrayType::Pointer yBounds);
-    void setZBounds(FloatArrayType::Pointer zBounds);
+
+    void setXBounds(FloatArrayType::Pointer xBnds);
+    void setYBounds(FloatArrayType::Pointer yBnds);
+    void setZBounds(FloatArrayType::Pointer zBnds);
```

Comment:
xBounds is a lot more readable than xBnds.  It's minor, but this probably shouldn't be changed.